product_warranty/views/product_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<odoo>
+    <record model="ir.ui.view" id="product_template_only_form_view">
+      <field name="name">product_template_form_view</field>
+      <field name="model">product.template</field>
+      <field name="inherit_id" ref="product.product_template_form_view"/>
+      <field name="arch" type="xml">
+        <xpath expr="//page[@name='general_information']" position="inside">
+            <group>
+                <field string="Warranty (Months)" name="warranty_time"/>
```

Comment:
You should put the string into the field. It will take it from the string attribute in Python

Reply:
You are right.